### PR TITLE
feat: Set automaticAuthentication to false [ROAD-1043]

### PR DESCRIPTION
**Pull request by asaf92**
### Description

Set LS to not automatically authenticate when a scan starts

### Checklist

- [ ] Tests added and all succeed
- [ ] Linted
- [ ] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### CODE REVIEW

File: src/snyk/common/languageServer/languageServer.ts [hunk 1]
```diff
@@ -33,8 +36,10 @@ export class LanguageServer implements ILanguageServer {
   async start(): Promise<void> {
     // TODO remove feature flag when ready
     if (!this.configuration.getPreviewFeatures().lsAuthenticate) {
+      await this.contextService.setContext(SNYK_CONTEXT.PREVIEW_LS_AUTH, false);
       return Promise.resolve(undefined);
```
Comment (j-luong) [inline src/snyk/common/languageServer/languageServer.ts:40]: You don't need to pass `undefined` in here,  it's implicit in `Promise.resolve()`

File: src/snyk/common/languageServer/settings.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { IConfiguration } from '../configuration/configuration';
 export type InitializationOptions = ServerSettings & {
   integrationName?: string;
   integrationVersion?: string;
+  automaticAuthentication?: string;
```
Comment (j-luong) [inline src/snyk/common/languageServer/settings.ts:7]: should this be a `bool`?

**Review by j-luong — APPROVED** [review]
some minor points, but other than that lgtm 👍

File: src/snyk/common/languageServer/settings.ts (hunk 2 above)
Comment (bastiandoetsch) [reply src/snyk/common/languageServer/settings.ts:7]: Nope, the backend expects a string afaik

File: src/snyk/common/languageServer/languageServer.ts [hunk 3]
```diff
@@ -90,6 +95,7 @@ export class LanguageServer implements ILanguageServer {
       ...settings,
       integrationName: CLI_INTEGRATION_NAME,
       integrationVersion: await Configuration.getVersion(),
+      automaticAuthentication: 'false',
```
[the remark this answers is not in the document]
Comment (asaf92) [reply src/snyk/common/languageServer/languageServer.ts:98]: All the other bool values are strings in the JSON objects

**Pull request merged**